src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -4,8 +4,11 @@ import { dateConverter } from '../../common/dateHelpers';
 import { timeConverter } from '../../common/timeHelpers';
 import { Button } from '../../common';
 import axios from 'axios';
+import { useOktaAuth } from '@okta/okta-react';
+import { getAuthHeader } from '../../../api/index';
 
 const ParentBookingCard = props => {
+  // child_name, child_id, course_id, description, end_date, end_time, instructor_id, instructor_name, location, prereqs, schedule_id, size, start_date ,start_time, subject
```

Comment:
Is this meant to be a temporary comment for you? If so remove it, if not could you explain the purpose of it?